cli/src/cmd/exec.rs

```diff
@@ -19,6 +21,7 @@ pub type Id = i64;
 pub type Hash = String;
 pub type ModuleName = String;
 pub type FunctionName = String;
+pub type FunctionInput = String;
```

Comment:
Since this is potentially a file _or_ data, can we consider something that can handle non UTF-8? May need a more complex type / enum to compose `Vec<u8>` and `PathBuf` and check if the first char of the vec is a `'@'`?

Reply:
Yeah good call!